doc(release) Release notes collect 24.10.10 (#4621)
centreon/centreon-documentation: versioned_docs/version-24.10/releases/centreon-os.mdx

## Centreon Monitoring Agent

### 24.10.10

Release date: `September 3, 2025`

<details open>
  <summary>Bug fixes</summary>

> - Fixed an issue where no output were returned from Centreon & custom plugins checks.

Reviewer: ```suggestion
- Fixed an issue where no output was returned from Centreon plugins and custom plugins.
```
